rllib/policy/tf_mixins.py

```diff
@@ -290,16 +290,19 @@ def __init__(
         action_space: gym.spaces.Space,
         config: TrainerConfigDict,
     ):
+        q_func_vars = self.model.trainable_variables()
+        target_q_func_vars = self.target_model.trainable_variables()
```

Comment:
sorry about the delay.
I took another careful look at this, and I feel like we actually don't have to change anything in this __init__()?
what is necessary is to simply change the q_func_vars(self) function and target_q_func_vars(self) function below to call trainable_variables() right?

sorry I didn't give the best suggestion in my first review.

also, what do you think we should do for the torch version of TargetNetworkMixin?

Thanks.

Reply:
@gjoliver I thought some other part of the code may be using the `policy.variables`, so did not touch that part. If no other code uses that, we can make the change in the respective methods, I'm not 100% sure about, can you please confirm that.

It seems for both DDPG and DQN, torch policy uses the same logic, in both of the cases it updates the entire `state_dict`.

Reply:
good question. our code really is a mess. we should have implemented get_states() and set_states() for the TargetNetworkMixIn instead, sigh.

can you do me a small favor then, just rename these 2 variables to:
trainable_q_func_vars and trainable_target_q_func_vars please.

Reply:
Thanks, updated the variables names.